src/components/articleDescription/ArticleDescription.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './articleDescription.scss';
+
+const ArticleDescription = (props) => {
+  const { description } = props;
+  return (
+    <div style={{ textAlign: 'center' }}>
```

Comment:
@Dubby20 any reason why we're using inline styles here as opposed to Bootstrap classes?

Reply:
Not really. I was hoping I would move it later but I've moved it now